autoware_adapi_v1_msgs/vehicle/msg/DoorStatus.msg

```diff
@@ -1,4 +1,3 @@
-# constants for door status
```

Comment:
I don't mind deleting them, but just out of curiosity, are there any reasons for removing this comment?

Reply:
This message is because there is only one variable, so the purpose of the constant is clear. This comment is used for messages containing multiple variables.